extensions/datacube/examples/example.json

```diff
@@ -1,5 +1,8 @@
 {
   "stac_version": "0.7.0",
+  "stac_extensions": [
```

Comment:
```
"stac_extensions": {
        "cube": "0.0.1"
    }
```

Reply:
See my comment below. I'm pretty sure we settled on arrays as the version information is basically included in the URL as the referenced schema is versioned.